src/libecalc/core/result/results.py

```diff
@@ -60,6 +60,8 @@ class GeneratorSetResult(GenericComponentResult):
 
     typ: Literal["genset"] = "genset"
     power_capacity_margin: TimeSeriesStreamDayRate
+    power_supply_onshore: Optional[TimeSeriesStreamDayRate]
+    max_usage_from_shore: Optional[TimeSeriesStreamDayRate]
```

Comment:
We should move it to LTP instead of having it in core as we haven't got the time to look into how to implement this the way we want it..in my opinion.